sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetIOSuite.scala

```diff
@@ -1461,10 +1462,7 @@ class ParquetIOSuite extends QueryTest with ParquetTest with SharedSparkSession
           vectorizedReader.initBatch(schema, partitionValues)
           vectorizedReader.nextKeyValue()
           val row = vectorizedReader.getCurrentValue.asInstanceOf[InternalRow]
-
-          // Use `GenericMutableRow` by explicitly copying rather than `ColumnarBatch`
-          // in order to use get(...) method which is not implemented in `ColumnarBatch`.
```

Comment:
Hmm, why it said `get` is not implemented in `ColumnarBatch`? Is this just out-dated?

Reply:
I think so, overtime `get` is implemented. cc original author of this test @HyukjinKwon

Reply:
But when used in partition column, `ColumnarBatchRow` is used instead of `GenericMutableRow` (i.e. `copy` won't be called). Is it correct?

Reply:
I thought so initially, but now I cannot find `ColumnarBatchRow.get` usages on `NullType` columns